### PR TITLE
isabelle-2022 -> isabelle-2023

### PR DESCRIPTION
**Pull request by rsoeldner**
## Description of changes

Update Isabelle 2022 to 2023 release.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/science/logic/isabelle/default.nix [hunk 1]
```diff
@@ -56,18 +56,18 @@ in stdenv.mkDerivation (finalAttrs: rec {
...
       fetchurl {
         url = "https://isabelle.in.tum.de/website-${dirname}/dist/${dirname}_linux.tar.gz";
-        sha256 = "1ih4gykkp1an43qdgc5xzyvf30fhs0dah3y0a5ksbmvmjsfnxyp7";
+        sha256 = "sha256-Go4ZCsDz5gJ7uWG5VLrNJOddMPX18G99FAadpX53Rqg=";
       }
     else
       fetchurl {
         url = "https://isabelle.in.tum.de/website-${dirname}/dist/${dirname}_linux_arm.tar.gz";
-        hash = "sha256-qI/BR/KZwLjnkO5q/yYeW4lN4xyUe78VOM2INC/Z/io=";
+        hash = "09s4cwygz465pmyqlf799a0mbbmlzhqj75xnk7raphqa92rp2g2g";
```
Comment (AndersonTorres) [inline pkgs/applications/science/logic/isabelle/default.nix:70]: How did you generate this?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/applications/science/logic/isabelle/default.nix (hunk 1 above)
Comment (rsoeldner) [reply pkgs/applications/science/logic/isabelle/default.nix:70]:
```
nix-prefetch-url https://isabelle.in.tum.de/website-Isabelle2023/dist/Isabelle2023_linux_arm.tar.gz
path is '/nix/store/5d85d8dhss9mja44kw7g8lrv16k8gsqh-Isabelle2023_linux_arm.tar.gz'
09s4cwygz465pmyqlf799a0mbbmlzhqj75xnk7raphqa92rp2g2g
```

same for 
```
nix-prefetch-url https://isabelle.in.tum.de/website-Isabelle2023/dist/Isabelle2023_macos.tar.gz
path is '/nix/store/my2ccy7ydcjhwi8d1avcfiz6hd5a185s-Isabelle2023_macos.tar.gz'
1x6pcngibmxsn27x1m6nmrm9gfz2ysg0j0l7jgzqsd675bcrcm6i
```

Comment (AndersonTorres) [reply pkgs/applications/science/logic/isabelle/default.nix:70]: It should be generated in SRI format.
How did you build this?
Have you read the OfBorg error messages?

Comment (rsoeldner) [reply pkgs/applications/science/logic/isabelle/default.nix]: @AndersonTorres moved to SRI format, building is done via: `nix build .#isabelle`

**Pull request closed without merging**